plasma_framework/test/src/transactions/PaymentTransactionModel.test.js

```diff
@@ -136,6 +134,38 @@ contract('PaymentTransactionModel', () => {
         const encoded = web3.utils.bytesToHex(transaction.rlpEncoded());
         await expectRevert(this.test.decode(encoded), 'Output amount must not be 0');
     });
+
+    describe('decodeOutput', () => {
```

Comment:
Minor: probably should move the two tests above inside here too:
- should fail when an output amount is zero
- should fail when an output type is zero